src/build.ts

```diff
@@ -405,6 +405,7 @@ async function _build(nitro: Nitro, rollupConfig: RollupConfig) {
   const buildInfo = {
     date: new Date(),
     preset: nitro.options.preset,
+    entry: nitro.options.entry.split("/").pop(),
```

Comment:
This is the **source entry** name.

Reply:
Yes, this is useful for presets like firebase, where one preset can have multiple entries : 
ie looking at the build output, we can't know if `firebase-gen-1` or `firebase-gen-2` is being used.

Reply:
I know that's a valid case but generally it doesn't seems to be best idea to add source entry to production info. It can be implemented per preset imo.

Reply:
That's a good point, what about renaming the preset if it's firebase ? Something like `firebase_${variant}` ?

Reply:
Renaming the user-input preset can also lead to more confusing behavior.